lib/addons/efs-csi-driver/index.ts

```diff
@@ -89,6 +89,8 @@ function populateValues(helmOptions: EfsCsiDriverProps, clusterName: string,
     setPath(values, "clusterName",  clusterName);
     setPath(values, "controller.serviceAccount.create",  false);
     setPath(values, "controller.serviceAccount.name",  serviceAccountName);
+    setPath(values, "node.serviceAccount.create",  false);
```

Comment:
What prompted this change? I recall vaguely there was an issue with the EFS SA, seems like the right change, just curious.

Reply:
EFS CSI Driver addon provisions two kinds of pods in the cluster: `controller` and `node`. Both of them requires IAM permissions to AWS resources. This line passes the same `k8s service account` to the `node` pods, which was missing, otherwise the addon fails to stabilize.